### PR TITLE
WT-6411 Added `firstObservedInHour` field in TimeRow

### PR DESCRIPTION
**Pull request by alvinllobrera**

### CODE REVIEW

**Review by tom-haines — COMMENTED** [review]
@alvinllobrera 
Suggest change to 
```
The firstObservedInHour represents the minute in the given activityHour when the activity was first observed. It is an integer value between 0 and 59.
```

**Review by vyshane — APPROVED** [review]
Nothing further to add

File: spec/openapi.yaml [hunk 1]
```diff
@@ -330,6 +330,11 @@ components:
         activityHour:
           type: integer
           description: The hour of the posted time in UTC. yyyyMMddhh
+        firstObservedInHour:
+          type: integer
+          description: >
+            The firstObservedInHour represents the minute in the activityHour when the activity was first observed.
+            It will be an integer value between 0 and 59.
         durationSecs:
           type: integer
```
Comment (vyshane) [inline spec/openapi.yaml:354]: Can you document `durationSecs` (and any other fields missing a description) while you're at it? See [posted time webhook doc](https://wisetime.io/docs/connect/posted-time-webhook/).

Comment (alvinllobrera) [reply spec/openapi.yaml:354]: Ok, done. Thanks for checking

**Pull request merged**